tools/Makefile

```diff
@@ -38,13 +38,7 @@ SET_VERSION_SCRIPTS = \
 	bin/gpssh-exkeys \
```

Comment:
gpssh-exkeys still in use. 'hawq ssh' will call it.
For others not in bin or sbin, there still plenty need to be remove, we will review and remove them in the future.